docs/rfcs/rfc-007-subscriptions.md

```diff
@@ -8,42 +8,42 @@ Our users would like to use [GraphQL Subscriptions](https://graphql.org/blog/sub
 
 ### Must have
 
-- To subscribe to the following events:
+- Subscribe to the following events:
   - CREATE
   - UPDATE
   - DELETE
-- Ability to filter which nodes are subscribed to (i.e. `where` clause) - at least by unique property
-- Ability to horizontally scale - either now, or able to do so without breaking in the future
-- Wherever possible, events are ordered
-- Events are fired individually
-- We won't mutate our users' databases - no metadata
-- All events are sent (for example, if node created and then deleted, we get both events)
-- Database transactions must be successful - no optimisticness  
-- Garbage collection of old subscriptions
-- Auth (on read)
+- Ability to filter which nodes are subscribed to (i.e. `where` clause).
+- Ability to horizontally scale - either now, or able to do so without breaking in the future.
+- Wherever possible, events are ordered.
+- Events are fired individually.
+- We won't mutate our users' databases - no metadata.
+- All events are sent (for example, if node created and then deleted, we get both events).
+- Database transactions must be successful - no optimisticness.
+- Garbage collection of old subscriptions.
+- Some form of auth validation.
```

Comment:
How about adding this?
- Shall not break web browser support of the `@neo4j/graphql` lib.

Reply:
Good point